yew/src/virtual_dom/mod.rs

```diff
@@ -343,14 +343,13 @@ impl Classes {
     /// Adds a class to a set.
     ///
     /// If the provided class has already been added, this method will ignore it.
-    pub fn push<T: Into<Self>>(&mut self, class: T) {
-        let classes_to_add: Self = class.into();
-        self.set.extend(classes_to_add.set);
+    pub fn push<T: Into<Cow<'static, str>>>(&mut self, class: T) {
+        self.set.insert(class.into());
     }
 
     /// Check the set contains a class.
-    pub fn contains<T: AsRef<str>>(&self, class: T) -> bool {
-        self.set.contains(class.as_ref())
+    pub fn contains<T: Into<Cow<'static, str>>>(&self, class: T) -> bool {
```

Comment:
This has much more overhead than `AsRef<str>`. I think we should keep it like it was before.

Reply:
Yes I know it's terrible